--- origsrc/gimp-2.10.0/configure.ac	2018-04-26 09:27:39.000000000 -0400
+++ src/gimp-2.10.0/configure.ac	2018-05-11 13:58:22.338397700 -0400
@@ -2041,15 +2041,22 @@ if test "x$enable_python" != xno; then
   # This might be able to be figured out from distutils, but it's
   # not documented so we won't rely on internal implementation
   PYLINK_LIBS=
-  if test "x$platform_win32" = "xyes"; then
+  case "$target_or_host" in
+  *-*-cygwin*)
+    PYBIN_PATH="$PYTHON"
+    PYLINK_LIBS="-L${py_exec_prefix}/lib/python${PYTHON_VERSION}/config -lpython${PYTHON_VERSION}"
+    ;;
+  *-*-mingw*)
     PYBIN_PATH="$py_exec_prefix\pythonw.exe"
     PYTHON_INCLUDES=`echo "$PYTHON_INCLUDES" | sed -e 's/\\\\/\\//g'`
     py_exec_prefix=`echo "$py_exec_prefix" | sed -e 's/\\\\/\\//g'`
     pylibversion=`echo $PYTHON_VERSION | sed -e 's/\\.//'`
     PYLINK_LIBS="-L${py_exec_prefix}/libs -lpython${pylibversion}"
-  else
+    ;;
+  *)
     PYBIN_PATH="$PYTHON"
-  fi
+    ;;
+  esac
   AC_SUBST(PYLINK_LIBS)
   AC_SUBST(PYBIN_PATH)
 
--- origsrc/gimp-2.10.0/plug-ins/pygimp/Makefile.am	2018-03-25 08:35:19.000000000 -0400
+++ src/gimp-2.10.0/plug-ins/pygimp/Makefile.am	2018-05-11 13:58:22.342401800 -0400
@@ -13,10 +13,9 @@ SUBDIRS = plug-ins
 
 if PLATFORM_WIN32
 no_undefined = -no-undefined
+endif
+if OS_WIN32
 shrext       = -shrext .pyd
-path_separator = ;
-else
-path_separator = :
 endif
 
 AM_CFLAGS = $(PYGIMP_EXTRA_CFLAGS)
@@ -149,7 +148,7 @@ pyinterpfile = $(pyinterpdir)/pygimp.int
 
 install-env-file:
 	$(mkinstalldirs) '$(DESTDIR)$(pyenvdir)'
-	echo '$(path_separator) PYTHONPATH=$${gimp_plug_in_dir}/$(pygimpbase)' > '$(DESTDIR)$(pyenvfile)'
+	echo '$(PATHSEP) PYTHONPATH=$${gimp_plug_in_dir}/$(pygimpbase)' > '$(DESTDIR)$(pyenvfile)'
 
 install-interp-file:
 	$(mkinstalldirs) '$(DESTDIR)$(pyinterpdir)'
